### ISSUE TITLE
Game completely freezing at random using SCT

### ISSUE THREAD

**Issue by SaiyanOfDarkness** [Artenuvielle/GW2-SCT#32]
It randomly happens when going into combat.  I am not sure if it's a certain skill icon that pops up that causes it, but each time I go to enter combat.. about a second later.. my game's screen just freeze frames.  I can still move my cursor around and if I right click and try to move my camera my cursor disappears, but the image on screen stays the same.  I have to Alt F4 out of my game and restart it.  Usually then after I restart it I'm good for a little while, and it doesn't do it for quite some time.

I am unsure why it is happening.  If it helps I am using the latest version of SCT, but I am using arcdps with suppress combat text enabled.

There is no crash log generated because of this as the game is completely froze forcing me to close the client via task manager.

**Comment by Artenuvielle** [comment]
Could you provide the log found at: gw2_install_dir/addons/sct/sct.log

**Comment by dwain-lab** [comment]
Yes, same thing happens to me here but then again, I am on old hardware so that might be a contributing factor.  FPS for me drops to like 9 but after the text scrolls are done it recovers.
